docs/sources/linking/linking-overview.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "Linking overview"
+type = "docs"
+keywords = ["grafana", "linking", "create links", "link panels", "link dashboards", "navigate"]
+[menu.docs]
+aliases = ["/docs/grafana/latest/features/navigation-links/"]
+identifier = "linking-overview"
+parent = "linking"
+weight = 100
++++
+
+# Linking overview
+
+You can use links to navigate between commonly-used dashboards or to connect others to your visualizations. Links let you create shortcuts to other dashboards, panels, and even external websites.
+
+Grafana supports dashboard links and panel links. Dashboard links are displayed at the top of the dashboard. Panel links are accessible by clicking an icon on the top left corner of the panel.
+
+## Which link should you use?
+
+Start by figuring out how you're currently navigating between dashboards. If you're often jumping between a set of dashboards and struggling to find the same context in each, links can help optimize your workflow.
+
+The next step is to figure out which link type is right for your workflow. Even though all the link types in Grafana are used to create shortcuts to other dashboards or external websites, they work in different contexts.
+
+- If the link relates to most if not all of the panels in the dashboard, use [dashboard links]({{< relref "dashboard-links.md" >}}).
+- If you want to drill down into specific panels, use [panel links]({{< relref "panel-links.md" >}}).
+- If you want to link to an external site, you can use either type of link.
```

Comment:
We are missing data links here (https://grafana.com/docs/grafana/latest/reference/datalinks/)